corehq/apps/reports/tasks.py

```diff
@@ -183,8 +184,7 @@ def monthly_reports():
 
 @periodic_task(run_every=crontab(hour=[22], minute="0", day_of_week="*"), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE','celery'))
 def saved_exports():
-    for group_config in HQGroupExportConfiguration.view("groupexport/by_domain", reduce=False,
-                                                        include_docs=True).all():
+    for group_config in get_all_hq_group_export_configs:
```

Comment:
This probably matters less. This is run once a day, so for quickcache to do anything it'd have to cache for days; and currently there are only 500 `HQGroupExportConfiguration` docs. If that number started growing for some reason, then it's probably growing at a rate faster than one per day, in which the cache would be invalid by the time it's hit again (since every save invalidates that cache). So the thing we'd probably have to do at that theoretical point at which loading it once a day is a problem would be something entirely different anyway.